apps/namadillo/package.json

```diff
@@ -27,6 +27,7 @@
     "jotai": "^2.6.3",
     "jotai-tanstack-query": "^0.8.5",
     "lodash.debounce": "^4.0.8",
+    "namada-chain-registry": "https://github.com/anoma/namada-chain-registry#add-housefire-ibc",
```

Comment:
can you confirm it'll work fine after the PR is merged and the branch is deleted? Maybe we should use the commit hash here

Reply:
Updated now to point to the main branch now that my PR there is merged.

Just out of interest, I checked the `yarn.lock` and it seems like it does point to a commit. But if the branch does get deleted I have no idea if GitHub keeps track of the commit.